### ISSUE TITLE
There is no notification about connected resources when removing a group

### ISSUE THREAD

**Issue by tdipisa** [geosolutions-it/geonode#547]
@giohappy, as reported by ADBPO and confirmed by @simboss through quick test in [GeoSolutions GeoNode DEV](https://dev.geonode.geo-solutions.it/), GeoNode doesn't inform the user if there are connected resources to a group while deleting it, there is only a simple  yes/no confirmation dialog.

**Comment by giohappy** [comment]
should GeoNode send them? I mean, we might agree that it would be useful, but I'm not aware of such a feature right now @tdipisa

**Comment by afabiani** [comment]
Opened an issue here

https://github.com/GeoNode/geonode/issues/6644

**Comment by simboss** [comment]
@afabiani I see you are already doing commits but this is a little premature since first we should discuss what behavior we expect.
This way of working makes it very difficult to know how things work and what you are doing, so please, follow the flow of work and discuss things before doing them.

**Comment by simboss** [comment]
@giohappy @afabiani I would expect geonode to tell me if I want to delete the related resources if I try to delete a group or a user.
If not, it should let me decide what to do with them, for example it should be possible to assign ownership to the admin or to another user of my group(s).

Feedback?

**Comment by giohappy** [comment]
I agree with @simboss. The removal of the CASCADE behaviour is fine in any case, but it's not enoigh from the user perspective , since he should at least be notified that the group (user, etc.) had resources assigned

**Comment by afabiani** [comment]
This requires changing the admin panel behavior. We need to open a feature request on GeoNode along with mockups.

**Comment by giohappy** [comment]
@simboss I can open a feature request for this, but let us know if we should work on it or leave it for the future.

**Comment by simboss** [comment]
@giohappy I would want at least to see a plan for this and an estimate.
This behavior has already bitten use once, I would learn from experience and avoid being bitten twice.

**Comment by simboss** [comment]
@afabiani why the admin panel?
This is something that the user should present to the users when they are deleting resources it is not just for the admin.

**Comment by giohappy** [comment]
@afabiani looking at commit [c6b14bf9e](https://github.com/GeoNode/geonode/commit/c6b14bf9e9ac47f10e6c69e0e7fc7f4d0c2a4d61) I see that for the moment you "solved" the issue by removing the cascade behaviour. This means that resources won't be deleted, and the group field is simply set to null. 

@tdipisa is this behaviour fine for ADBPO? Can we close this issue?

**Comment by tdipisa** [comment]
@giohappy the current behavior is fine for the ADBPO use case at the moment. Anyway, I totally agree with [this comment](https://github.com/geosolutions-it/geonode/issues/547#issuecomment-727859596) and the posibility to transfer the ownership or at least a warning to inform the user would be useful for the user perspective. I think this can be considered as a general improvement of GeoNode, I don't know if you want to keep track of this with a new issue for the future instead of using this issue.

**Comment by giohappy** [comment]
@tdipisa can we close this?

**Comment by simboss** [comment]
I am not too happy with the current hack.

Let's rediscuss.
